src/govuk/helpers/helpers.test.js

```diff
@@ -1,40 +1,58 @@
-const glob = require('glob')
-const path = require('path')
+const { join } = require('path')
 
 const sassdoc = require('sassdoc')
 
+const { getListing } = require('../../../lib/file-helper')
 const { renderSass } = require('../../../lib/jest-helpers')
 const configPaths = require('../../../config/paths.js')
 
-const sassFiles = glob.sync(`${configPaths.src}/helpers/**/*.scss`)
-
 describe('The helpers layer', () => {
+  let sassFiles
+
+  beforeAll(async () => {
+    sassFiles = await getListing(configPaths.src, 'helpers/**/*.scss', {
+      ignore: ['**/_all.scss']
+    })
+  })
+
   it('should not output any CSS', async () => {
-    const helpers = path.join(configPaths.src, 'helpers', '_all.scss')
+    const helpers = join(configPaths.src, 'helpers', '_all.scss')
 
     const output = await renderSass({ file: helpers })
     expect(output.css.toString()).toEqual('')
   })
 
-  it.each(sassFiles)('%s renders to CSS without errors', (file) => {
-    return renderSass({ file })
+  it('renders CSS for all helpers', () => {
+    const sassTasks = sassFiles.map((sassFilePath) => {
+      const file = join(configPaths.src, sassFilePath)
+
+      return expect(renderSass({ file })).resolves.toEqual(
+        expect.objectContaining({
+          css: expect.any(Object),
+          stats: expect.any(Object)
+        })
```

Comment:
*question*: This checks the shape of the return of `renderSass`, which we can trust Sass' `.render` to do properly. As [the method throws a `LegacyException` if the compilation fail](https://sass-lang.com/documentation/js-api/modules#render), and in turn fail `Promise.all`, wouldn't it be enough to no check what's returned and have that a test that fails on exception rather than unmet expectations (which will always be met)?

Reply:
@romaricpascal Probably, I'm just matching these tests with the same format we use here:
https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/components.test.js#L29

We can change them all in another PR if it's a concern?

Reply:
Ah, I didn't see that file had that structure for this test. Happy with aligning them all to do the same then 😄